packages/server/src/oauth/utils.test.ts

```diff
@@ -430,6 +433,223 @@ describe('OAuth utils', () => {
     }
   });
 
+  describe('getAuthTokens with email scope', () => {
+    test('Access token includes email when email scope is requested for User', async () => {
+      await withTestContext(async () => {
+        const systemRepo = getSystemRepo();
+        const { project } = await createTestProject();
+
+        // Create a User with email
+        const userEmail = `test-${randomUUID()}@example.com`;
+        const user = await systemRepo.createResource<User>({
+          resourceType: 'User',
+          email: userEmail,
+          firstName: 'Test',
+          lastName: 'User',
+        });
+
+        // Create a Patient profile
+        const patient = await systemRepo.createResource<Patient>({
```

Comment:
Seems like this should be made with a repo associated with `project` so that you don't have to specify `meta` at all. You can easily get that repo by modifying up above to `createTestProject({withRepo: true})` and pulling repo out of the result.